users.go

```diff
@@ -0,0 +1,69 @@
+package unifi
+
+import (
+	"fmt"
+	"strings"
+)
+
+// GetUsers returns a response full of clients that connected to the UDM within the provided amount of time
+// it uses the insight historical connections data set
+func (u *Unifi) GetUsers(sites []*Site, hours int) ([]*User, error) {
+	data := make([]*User, 0)
+
+	for _, site := range sites {
+		var response struct {
+			Data []*User `json:"data"`
+		}
+		params := fmt.Sprintf(`{ "type": "all:", "conn": "all", "within":%d }`, hours)
```

Comment:
Either add parens to `var` and put `params` into it, or add a white space and uncuddle `params` from the `var`. Sorry for the nitpick comments. These are all in the build output. https://app.travis-ci.com/github/unpoller/unifi/builds/235303661